docs/userGuide/syntax/embeds.md

```diff
@@ -56,4 +56,4 @@ Embedded slide deck:
 
 @[powerpoint](https://onedrive.live.com/embed?cid=A5AF047C4CAD67AB&resid=A5AF047C4CAD67AB%212070&authkey=&em=2)
 
-</div>
\ No newline at end of file
+</div>
```

Comment:
Not sure if there's an unnecessary white space change or something here?

Reply:
Seems to be an EOL character though I'm not sure what caused it. Fixed!